res/css/views/auth/_LoginWithQR.pcss

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_LoginWithQRSection .mx_AccessibleButton {
+    margin-right: 10px;
+}
+
+.mx_AuthPage .mx_LoginWithQR {
+    .mx_AccessibleButton {
+        display: block !important;
+    }
+
+    .mx_AccessibleButton + .mx_AccessibleButton {
+        margin-top: 8px;
+    }
+
+    .mx_LoginWithQR_separator {
+        display: flex;
+        align-items: center;
+        text-align: center;
+
+        &::before, &::after {
+            content: '';
+            flex: 1;
+            border-bottom: 1px solid $quinary-content;
+        }
+
+        &:not(:empty) {
+            &::before {
+                margin-right: 1em;
+            }
+            &::after {
+                margin-left: 1em;
+            }
+        }
+    }
+
+    font-size: $font-15px;
+}
+
+.mx_UserSettingsDialog .mx_LoginWithQR {
+    .mx_AccessibleButton + .mx_AccessibleButton {
+        margin-left: 12px;
+    }
+
+    font-size: $font-14px;
+
+    h1 {
+        font-size: $font-24px;
+        margin-bottom: 0;
+    }
+
+    li {
+        line-height: 1.8;
+    }
+
+    .mx_QRCode {
+        padding: 12px 40px;
+        margin: 26px 0;
+    }
+
+    .mx_LoginWithQR_buttons {
+        text-align: center;
+    }
+
+    .mx_LoginWithQR_qrWrapper {
+        display: flex;
+    }
+}
+
+.mx_LoginWithQR {
+    min-height: 350px;
+    display: flex;
+    flex-direction: column;
+
+    .mx_LoginWithQR_centreTitle {
+        h1 {
+            text-align: centre;
+        }
+    }
+
+    h1 > svg {
+        &.normal {
+            color: $secondary-content;
+        }
+        &.error {
+            color: $alert;
+        }
+        &.success {
+            color: $accent;
+        }
+        height: 1.3em;
+        margin-right: 8px;
```

Comment:
This and other paddings and margins in the file should use our `$spacing-*` variables where possible